docs/user_guide/advanced_composition/conditionals.md

```diff
@@ -124,6 +124,25 @@ You can run the workflow locally as follows:
 :lines: 181-188
 ```
 
+## Running a noop task in a conditional
+
+In some cases, you may want to skip the execution of a conditional workflow if a certain condition is not met.
+You can achieve this by using the `echo` task, which simply returns the input value.
+
+:::{note}
+To enable the echo plugin in the backend, add the plugin to Flyte's configuration file.
+```yaml
+task-plugins:
+  enabled-plugins:
+    - echo
+```
+:::
+
+```{rli} https://raw.githubusercontent.com/flyteorg/flytesnacks/master/examples/advanced_composition/advanced_composition/conditional.py
```

Comment:
I recommend using a link with a SHA instead, since it will be more stable than a link to the `master` branch: https://raw.githubusercontent.com/flyteorg/flytesnacks/89bf7bc7788802097904c5f9ffb75ba70ef980a6/examples/advanced_composition/advanced_composition/conditional.py

Reply:
no problem! thank you so much!